.circleci/config.yml

```diff
@@ -33,9 +44,16 @@ commands:
 
 jobs:
   Build Tests:
+    parameters:
+      device:
+        type: string
+        description: The device (e.g. "iPhone 11") to use when compiling the build.
+      ios-version:
+        description: The iOS deployment target (e.g. "13.0") used to compile the build that will be used for testing.
+        type: string
```

Comment:
I know this is not for this PR to fix, but just putting this here for posterity, and for some feedback when we do a similar update on other iOS repos: since it seems we already use YAML aliases & references for those (on line 17-26), which I love :+1:, we might as well consider DRY-ing those parameters with aliases & references too 😉 

i.e. just before line 20, declare this:
```yaml
device_and_os_params: &device_and_os_params
  device:
    type: string
    description: The device (e.g. "iPhone 11") to use when compiling the build.
  ios-version:
    description: The iOS deployment target (e.g. "13.0") used to compile the build that will be used for testing.
    type: string
```
And then here we can use `parameters: *device_and_os_params` (or `<<: *device_and_os_params` merge operation if we need more params in addition to those 2)

Again, not for this PR to fix, but worth noting for next time we address this.